model/feature_flags.go

```diff
@@ -75,6 +75,8 @@ type FeatureFlags struct {
 	PlanUpgradeButtonText string
 
 	PostPriority bool
+
+	EnableVoiceMessages bool
```

Comment:
we have name of feature flags without prefixes. can this also be just VoiceMessage